src/views/LoanDetails/LoanDetails.test.js

```diff
@@ -265,228 +266,280 @@ const propsData = {
 
 const renderLoanProxyDetails = (props1) => renderWithRouter(<LoanDetails {...props1} />);
 
-describe('Render LoanProxyDetails component', () => {
-  it('When props ID and proxy ID are same with Loan Missing', () => {
-    mockGetLodash.mockReset();
-    mockGetLodash.mockImplementation(() => {
-      return 'Lost and paid';
+describe('LoanDetails', () => {
+  describe('Render LoanProxyDetails component', () => {
+    it('When props ID and proxy ID are same with Loan Missing', () => {
```

Comment:
There are a lot of tests that existed before this PR that don't follow the describe/it structure. These `it` statements don't tell what the expected behavior is.
I understand that it's out of scope of the ticket to refactor tests completely, so could you just fix the structure a bit? Wrap `it` statements in `describe`. `describe` should have the pre-conditions explained and `it` should have the expected behaviour?

Reply:
I see there are many tests that are not properly structured.
I think it will be good to address them in separate ticket, in order to keep the purpose of this PR in the scope of the ticket. Is that fine with you?

Reply:
Sure, please create a ticket then and it can be worked on separately

Reply:
Hey @BogdanDenis 
Created a ticket for the same - https://issues.folio.org/browse/UIU-3001
Please review and adjust any details in you wish to. Thank you